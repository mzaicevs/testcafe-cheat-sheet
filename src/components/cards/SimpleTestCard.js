import React from "react";
import { simpleTest } from "../../snippets";
import CodeCard from "../../base_components/CodeCard";

const SimpleTestCard = () => (
  <CodeCard
    cardTitle={simpleTest.title}
    codeSnippet={{
      code: simpleTest.code,
      language: simpleTest.language
    }}
  />
);

export default SimpleTestCard;
