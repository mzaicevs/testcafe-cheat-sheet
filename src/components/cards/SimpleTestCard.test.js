import React from "react";
import { shallow } from "enzyme";
import SimpleTestCard from "./SimpleTestCard";

describe("SimpleTestCard", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<SimpleTestCard />);
    expect(wrapper).toMatchSnapshot();
  });
});
