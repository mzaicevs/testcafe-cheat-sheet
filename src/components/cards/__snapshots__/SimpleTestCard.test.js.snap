// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleTestCard matches snapshot 1`] = `
<CodeCard
  cardTitle="Simple Test"
  codeSnippet={
    Object {
      "code": Array [
        "import { Selector } from 'testcafe';
    
    fixture \`Getting Started\`
      .page \`http://devexpress.github.io/testcafe/example\`;
                
    test('My first test', async t => {
      // Test code
    });",
      ],
      "language": "javascript",
    }
  }
/>
`;
